pkg/logging/blobstorage/root.go

```diff
@@ -0,0 +1,28 @@
+package blobstorage
+
+import (
+	"io"
+
+	"github.com/fastly/cli/pkg/common"
+	"github.com/fastly/cli/pkg/config"
+)
+
+// RootCommand is the parent command for all subcommands in this package.
+// It should be installed under the primary root command.
+type RootCommand struct {
+	common.Base
+	// no flags
+}
+
+// NewRootCommand returns a new command registered in the parent.
+func NewRootCommand(parent common.Registerer, globals *config.Data) *RootCommand {
+	var c RootCommand
+	c.Globals = globals
+	c.CmdClause = parent.Command("blobstorage", "Manipulate Fastly service version Azure Blob Storage logging endpoints")
```

Comment:
It would be good if we can keep consistency with the API endpoints, as `blobstorage` is too generic:
```suggestion
	c.CmdClause = parent.Command("azureblob", "Manipulate Fastly service version Azure Blob Storage logging endpoints")
```

Reply:
> It would be good if we can keep consistency with the API endpoints, as `blobstorage` is too generic:

@phamann per you review comment, do you think everything (i.e., package, filenames, etc.) should be renamed? (I would prefer everything).

The only reason that I didn't was because it is just "BlobStorage" in [go-fastly](https://github.com/fastly/go-fastly/blob/master/fastly/blobstorage.go).